FirebaseRemoteConfigSwift/Tests/SwiftAPI/FirebaseRemoteConfigSwift_APIBuildTests.swift

```diff
@@ -16,14 +16,15 @@ import XCTest
 
 import FirebaseCore
 import FirebaseRemoteConfig
+import FirebaseRemoteConfigInterop
 import FirebaseRemoteConfigSwift
 
 final class FirebaseRemoteConfigSwift_APIBuildTests: XCTestCase {
   func usage() throws {
     // MARK: - FirebaseRemoteConfig
 
     // TODO(ncooke3): These global constants should be lowercase.
-    let _: String = FirebaseRemoteConfig.NamespaceGoogleMobilePlatform
```

Comment:
This test should remain since it's still part of the public API

Reply:
cc @danasilver, @ddnan

Reply:
Working on a fix

Reply:
Fixed in https://github.com/firebase/firebase-ios-sdk/pull/12503